flink-runtime/src/test/java/org/apache/flink/runtime/io/network/netty/DummyConnectionManager.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.runtime.io.network.netty;
+
+import org.apache.flink.runtime.io.network.ConnectionID;
+import org.apache.flink.runtime.io.network.ConnectionManager;
+import org.apache.flink.runtime.io.network.TaskEventPublisher;
+import org.apache.flink.runtime.io.network.partition.ResultPartitionProvider;
+
+import org.apache.flink.shaded.netty4.io.netty.channel.embedded.EmbeddedChannel;
+
+import java.io.IOException;
+
+import static org.apache.flink.runtime.io.network.partition.InputChannelTestUtils.STUB_CONNECTION_ID;
+
+/**
+ * A dummy implementation of the {@link ConnectionManager} which is mainly used for tests
+ * to avoid mock {@link PartitionRequestClient}.
+ */
+public class DummyConnectionManager implements ConnectionManager {
```

Comment:
I would suggest to call it `TestingConnectionManager` which is a bit more in line with recent additions.